int32 robot_id
float32[] path_x
float32[] path_y
float32[] path_z
bool success